### ISSUE TITLE
[bug] issues about pca_plot function

### ISSUE THREAD

**Issue by YTLogos** [almeidasilvaf/HybridExpress#1]
Hi @almeidasilvaf,

Thank you for developing such a fantastic package! I've encountered an issue while using the `pca_plot` function. Here's the code I ran:

```
pca_plot(se_c, color_by = "Generation", add_mean = TRUE) + 
  labs(title = "PCA of samples")
```

However, it returned the following error:

```
Error in if (!column %in% names(cdata)) { : length is zero
```

After reviewing the code, I believe the issue might stem from the `check_coldata_column` function:

```
check_coldata_column <- function(se, column) {
    
    cdata <- colData(se)
    if(!column %in% names(cdata)) {
        stop("Column '", column, "' is not present in the `colData` slot.")
    }
    
    return(TRUE)
}
```

It seems that when `shape_by = NULL`, the function checks for NULL in `names(cdata)`, which triggers the error.

Would it be possible to modify the code to handle this case properly?

Thanks again for your hard work!

Best regards,
Tom

**Comment by almeidasilvaf** [comment]
Hi, @YTLogos 

Thanks a lot for your feedback and for providing a clear description of the issue. This has now been fixed in ec9fd9852b8567e3120e49c685b6483aae96d2c6.

You can install the latest version of __HybridExpress__ with:

```r
# From Bioconductor
BiocManager::install("HybridExpress")

# From GitHub
BiocManager::install("almeidasilvaf/HybridExpress")
```

Thank you for using __HybridExpress__, and let me know if you have other issues. I will close this issue.

Best,
Fabricio

**Comment by YTLogos** [comment]
Hi @almeidasilvaf,

Thank you so much for the quick fix! I've tested the updated version, and it works perfectly now. I really appreciate your responsiveness and support.

Best regards,
Tom
